geoportailv3/static/js/catalog/layertree.html

```diff
@@ -1,5 +1,5 @@
-<ul id="catalog-{{::layertreeCtrl.uid}}" class="catalog">
```

Comment:
The `uid` of the `NgeoLayertreeController` won't change so I'm not sure why you need to remove one-time binding for `layertreeCtrl.uid`?